### ISSUE TITLE
TS does not compile .ts files

### ISSUE THREAD

**Issue by wzup** [microsoft/TypeScript#32326]
### TS doesn't compile files

**TypeScript Version:**

```
  "dependencies": {
    "@types/react": "^16.8.22",
    "@types/react-dom": "^16.8.4",
    "typescript": "^3.5.3"
  }
```

**Search Terms:**

Typescript does not compile

**Code**

My **tsconfig.json**

```json
{
  "compilerOptions": {
    "target": "es5",
    "lib": [
      "dom",
      "dom.iterable",
      "esnext"
    ],
    "allowJs": true,
    "skipLibCheck": true,
    "esModuleInterop": true,
    "allowSyntheticDefaultImports": true,
    "strict": true,
    "forceConsistentCasingInFileNames": true,
    "module": "esnext",
    "moduleResolution": "node",
    "resolveJsonModule": true,
    "isolatedModules": false,
    "noEmit": true,
    "sourceMap": true,
    "baseUrl": "./",
    "jsx": "preserve"
  },
  "include": [
    // All these options do not do the job 
    "./*.ts",
    "./*",
    "*",
    "./"
  ],
  "exclude": [
    "node_modules",
    "**/*.spec.ts"
  ]
}
```

My file to compile, **sample.ts**
```ts
// sample.ts
const f = (a: string, b: number) => a + b;
```

My **package.json**
```json
// package.json
{
  "name": "ts2",
  "version": "1.0.0",
  "description": "Typescript playground",
  "main": "index.js",
  "scripts": {
    "test": "echo \"Error: no test specified\" && exit 1",
    "tsc": "./node_modules/.bin/tsc"
  },
  "author": "",
  "license": "ISC",
  "dependencies": {
    "@types/react": "^16.8.22",
    "@types/react-dom": "^16.8.4",
    "typescript": "^3.5.3"
  }
}
```

I execute `tsc` to compile:
```bash
$ npm run tsc
``` 

**Expected behavior:**

Typescript compiles my file and creates `sample.js`

**Actual behavior:**

Nothing happens. `.js` version is not created.

```bash
$ npm run tsc

> ts2@1.0.0 tsc /Users/sergey/sbox/ts2
> tsc
```

**Playground Link:** 

Repo to reproduce 

https://github.com/wzup/ts-bug

**Related Issues:**

**Comment by IllusionMH** [comment]
You shouldn't use `"noEmit": true` in your `tsconfig.json` if you compile with `tsc`

From compiler options description `--noEmit` - Do not emit outputs.

**Comment by wzup** [comment]
@IllusionMH 
Yes, this fixes the issue.
Thanks

**Comment by RyanCavanaugh** [comment]
It looks like this is a question rather than a bug report. This issue tracker is for tracking bugs and active work on TypeScript *itself*, rather than a general forum for programmers using TypeScript to get help or ask questions.

You can ask questions on sites like [Stack Overflow](https://stackoverflow.com/questions/tagged/typescript). We are not able to provide one-on-one support on the issue tracker. Please read the issue template carefully - it has important information on what kinds of reports can be acted on here, as well as links to useful TypeScript resources. Thanks!

**Comment by arslan7172** [comment]
tsc.cmd -v
tsc.cmd --init

**Comment by dcow** [comment]
It's probably filed as a bug because the author thought it was a bug. I would too if I ran `tsc` and nothing happened. In fact I've spent the last hour trying to figure out why `tsc` isn't emitting files when I run it. Might be worth considering some type of output to `stderr` indicating that `tsc` will not emit output since `noEmit` is set to `true`. At least then people wouldn't waste time banging their head.
